### PR TITLE
Support --fmt

### PR DESCRIPTION
**Pull request by ry**
Depends on #1648

@kt3k

### CODE REVIEW

**Comment by ry** [comment]
Hitting an issue with an invalid Utf8 char causing V8 to crash (debug build only)

stack trace and the data here https://gist.github.com/ry/f53636889f29e5bc9463b6828d910b72

**Comment by ry** [comment]
Yes, I guess so. I will still put it off a bit longer - I want to experiment with only flags for a bit.

File: src/main.rs [hunk 1]
```diff
@@ -80,6 +80,11 @@ fn main() {
     LevelFilter::Warn
   });
 
+  if flags.fmt {
+    rest_argv.insert(1, "https://deno.land/x/std/prettier/main.ts".to_string());
+    flags.allow_write = true;
```
Comment (piscisaureus) [inline src/main.rs]: a bit hacky...

File: tools/prefetch_test.py [hunk 2]
```diff
@@ -2,20 +2,15 @@
...
-import tempfile
+from util import mkdtemp, tests_path, run_output, green_ok
 import shutil
 
 
 def prefetch_test(deno_exe):
     sys.stdout.write("prefetch_test...")
     sys.stdout.flush()
 
-    # On Windows, set the base directory that mkdtemp() uses explicitly. If not,
```
Comment (piscisaureus) [inline tools/prefetch_test.py]: Why was this rolled back? I don't think the underlying issue has been fixed.

**Pull request merged**